pkg/flowkit/services/services.go

```diff
@@ -61,3 +61,17 @@ func NewServices(
 		Tests:        NewTests(state, logger),
 	}
 }
+
+func (s *Services) SetLogger(logger output.Logger) {
```

Comment:
This seems slightly fragile. Is there a different way to do this?

Maybe all of the "components" (s.Transactions, s.Scripts, ...) should have a common interface and be in a list, over which you can iterate and set the log.

Reply:
I plan to refactor and do just that, didn't want to also do that in this PR tho, but there's an upcoming refactor happening soon improving bunch of ugly stuff like this.